server/app.js

```diff
@@ -9,6 +9,17 @@ const pingRouter = require("./routes/ping");
 
 const { json, urlencoded } = express;
 
+//Set up mongoose connection
+const mongoose = require("mongoose");
+const mongoDB = process.env.DB_URI;
+mongoose.connect(mongoDB, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+  useFindAndModify: false,
+});
+var db = mongoose.connection;
+db.on("error", console.error.bind(console, "MongoDB connection error:"));
+
 var app = express();
```

Comment:
Let's make sure we're consistently using let/const throughout the project. For these, const would be preferred since they shouldn't change.

Reply:
Okay fixed those.